src/ai/ai_core.cpp

```diff
@@ -293,13 +294,17 @@
 
 /* static */ int AI::GetStartNextTime()
 {
-	/* Find the first company which doesn't exist yet */
-	for (CompanyID c = COMPANY_FIRST; c < MAX_COMPANIES; c++) {
-		if (!Company::IsValidID(c)) return AIConfig::GetConfig(c, AIConfig::SSS_FORCE_GAME)->GetSetting("start_date");
+	int start_next = _settings_game.ai.ai_start_next;
+	if (start_next > 0 && AI::START_NEXT_DEVIATION != 0) {
```

Comment:
Useless !=0 check as `AI::START_NEXT_DEVIATION` is always 60

Reply:
I see, I was trying to be thorough. The equivalent code had those checks in place. As long as no one changes the defaults, I can remove the useless check.